src/protocol/malicious.rs

```diff
@@ -156,7 +156,10 @@ impl<F: Field> SecurityValidator<F> {
     /// ## Panics
     /// Will panic if the mutex is poisoned
     #[allow(clippy::await_holding_lock)]
-    pub async fn validate(self, ctx: ProtocolContext<'_, F>) -> Result<(), BoxError> {
+    pub async fn validate(
+        self,
+        ctx: ProtocolContext<'_, Replicated<F>, F>,
```

Comment:
It is because it calls `reveal` and `check_zero` protocols that currently support `Replicated` only.
once they are generic over any secret share type, we can change this code

Reply:
Open an issue for that?

Reply:
#201